gef.py

```diff
@@ -6772,11 +6772,10 @@ def do_invoke(self, _: List[str], **kwargs: Any) -> None:
 
         args : argparse.Namespace = kwargs["arguments"]
         if args.registers and args.registers[0]:
-            requested_regs = set(args.registers)
-            valid_regs = set(gef.arch.all_registers) & requested_regs
-            if valid_regs:
-                regs = valid_regs
-            invalid_regs = requested_regs - valid_regs
+            all_regs = gef.arch.all_registers
+            requested_regs = args.registers
+            regs = [reg for reg in requested_regs if x in all_regs]
+            invalid_regs = [reg for reg in requested_regs if x not in all_regs]
```

Comment:
this can use set logic

Reply:
```suggestion
            invalid_regs = set(requested_regs) - set(all_regs)
```

Reply:
Without evidence that the process of turning into a set and then doing set operations is noticeably more performant, it doesn't seem to be worth changing to a `set` just so we have a `set`. It doesn't feel like adds any clarity and I suspect it doesn't add any performance either.

Reply:
Let me be the one who decides that.